src/cli/self_update/unix.rs

```diff
@@ -145,8 +145,8 @@ pub fn self_replace() -> Result<utils::ExitCode> {
     Ok(utils::ExitCode(0))
 }
 
-fn remove_legacy_paths() -> Result<()> {
-    let export = format!("export PATH=\"{}/bin:$PATH\"\n", shell::cargo_home_str()?).into_bytes();
+fn remove_legacy_path(path: String) -> Result<()> {
+    let export = path.into_bytes();
```

Comment:
`path` is not actually a path but a line, so I would rename both the function (`remove_line`) and the parameter (`line`?).
The line is also not always an "export" line.

Reply:
Renamed the function to `remove_legacy_source_command` and the argument to `source_cmd`.  Finally renamed `export` as `cmd_bytes` - All very good points, thank you for taking the time to review this for me.